icechunk-python/python/icechunk/_icechunk_python.pyi

```diff
@@ -239,6 +241,9 @@ class S3Credentials:
     class Static:
         def __init__(self, _0: S3StaticCredentials) -> None: ...
 
+    class Refreshable:
+        def __init__(self, _0: bytes) -> None: ...
```

Comment:
Can we call this something other than `_0` hehe? `pickled_function`?

Reply:
good catch

Reply:
done